profile/src/test/resources/test-profiles/invalid/allof-misspelled.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
These profiles (in the `invalid` folder) must only have a single error, to keep them useful. The type should be added to make it valid otherwise. This comment applies to most of the invalid files here.

Reply:
Currently the tests aren't checking for that so they are basically pointless right now. I'm hoping to deal with that under #1329

Reply:
You're right, it would be good to have some confidence that there is only one mistake, but I think the tests are currently useful because they currently only have one mistake, but these changes are adding another mistake, so really they should be updated (or removed altogether if we don't think they are useful)